src/index.js

```diff
@@ -11,7 +11,7 @@ import {StaticShadows} from './props.shadows.js'
 import SVG from './props.svg.js'
 import Zindex from './props.zindex.js'
 
-export default OpenProps = {
+export default {
```

Comment:
this change a suggestion because it wasnt doing anything meaningful? or you needed to change it to fix something?

Reply:
Fix. It's an assignment to a non-existent variable. The alternative is:

```javascript
const OpenProps = { ... };
export default OpenProps;
```